src/issues/issueFeatureRegistrar.ts

```diff
@@ -255,7 +255,13 @@ export class IssueFeatureRegistrar implements vscode.Disposable {
 	}
 
 	async createIssue() {
-		return this.makeNewIssueFile(vscode.window.activeTextEditor?.document.uri);
+		let uri = vscode.window.activeTextEditor?.document.uri;
+		if (!uri) {
+			uri = (await this.chooseRepo('Select the repo to create the issue in.'))?.repository.rootUri;

```

Comment:
Should a picker still be shown if there's only one available repo?

Reply:
No, it should not. Thanks for catching! I have updated chooseRepo to return the first repo if there is only one.